client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -62,9 +64,22 @@ class HandleEmailedLinkForm extends React.Component {
 		showMagicLoginLinkExpiredPage: PropTypes.func.isRequired,
 	};
 
-	state = {
-		hasSubmitted: false,
-	};
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			hasSubmitted: false,
+		};
```

Comment:
Just curious, why not declaring the state outside of the constructor?

Reply:
No good reason -- I'll move it back.